docs/releases/Embeddinator-4000_0.1.md

```diff
@@ -0,0 +1,52 @@
+# Embeddinator-4000 0.1
+
+This is the first public release of Embeddinator-4000. It supports the Objective-C language on both macOS or iOS. More languages and platforms will be added in future releases.
+
+
+Requirements
+============
+
+* macOS 10.12 (Sierra) or later;
+* Xcode 8.3.2 or later;
+* Mono 5.0 or later;
+
+Optional
+--------
+
+* [Xamarin Studio](https://developer.xamarin.com/guides/cross-platform/xamarin-studio/); or 
+* [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/);
+* **Xamarin.iOS 10.11 preview** Available from the alpha channel of XS or VSfM
```

Comment:
Maybe spell out `XS` and `VSfM`, to us they're obvious abbreviations, to customers they may not be.